pkg/cmd/auth/login/login.go

```diff
@@ -59,6 +59,8 @@ func NewCmdLogin(f *cmdutil.Factory, runF func(*LoginOptions) error) *cobra.Comm
 		Long: heredoc.Docf(`
 			Authenticate with a GitHub host.
 
+			The %[1]shostname%[1]s is where you log in to GitHub. The default hostname is %[1]sgithub.com%[1]s.
```

Comment:
"log in" might be a touch confusing but I don't feel that strongly.

Reply:
Do you have a better word/phrase in mind?

Reply:
Not really. Maybe...

```
The default hostname is github.com, and can be overridden with the hostname flag
```

Maybe just leave it up to the reader to infer the URL?

Reply:
Let me ask a better question: why do you think this might be a touch confusing?

Reply:
This is where I might log into GitHub https://github.com/login/

Reply:
No worries. It does make sense... 🤔

Reply:
Given this, I like your suggestion. Adopted 👍